CopilotKit/packages/react-core/src/hooks/use-copilot-error.ts

```
@@ -0,0 +1,185 @@
+ *   });
+ *
+ *   return <form onSubmit={handleSubmit}>...</form>;
+ * }
+ * ```
+ */
+export function useCopilotErrorBoundary(componentName?: string) {
+  const { handleError } = useCopilotError(componentName);
+
+  const wrapAsync = useCallback(
+    <T extends any[], R>(fn: (...args: T) => Promise<R>) => {
+      return async (...args: T): Promise<R | void> => {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)